bycycle/plts/features.py

```diff
@@ -0,0 +1,73 @@
+"""Plot cycle features."""
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+###################################################################################################
+###################################################################################################
+
+
+def plot_feature_hist(df, burst_param, ax=None, bins='auto', **kwargs):
+    """ Plot a histogram for a cycle feature.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame or 1d array
+        Dataframe output from :func:`~.compute_features` or a 1d array.
+    burst_param : str
+        Column name of the parameter of interest in ``df``.
+    ax : matplotlib.Axes, optional, default: None
```

Comment:
I have tended to consistently put `ax` as the last argument (except for kwargs), and I think we might as well keep this up (unless you disagree). If it sounds good, check and update across all plot funcs.